### PR TITLE
Fix issue where channel upload viewcounts were creating a new line

### LINKED ISSUE #7078: improve wrapping for views + date on channel pages

### PR DESCRIPTION
**Pull request by mayeaux**
Fixes #7078

This address specifically 'large' view of over 1600px, where the worst offending was happening, but should also fix issues on other sizes as well, if anyone sees the view counts creating a third line on channel overview let me know and I'll take a look.

### CODE REVIEW

**Review by infinite-persistence — COMMENTED** [review]
- The difference in margin feels incoherent in a glance with sibling components (title).  Makes it too cramped to the left side?  
    - image
    - image
- The channel-stake-indicator gets cut off.
    - image
- Adding the tweak for specific layouts does cover most cases in English (nice), but even in other layouts it will still reflow:
    - image
- Other languages still would reflow after the tweak (although I understand we are just trying to cover the most common cases):
    - image

YT does it by tweaking tile count, and ultimately just reflow.

File: ui/component/fileViewCountInline/view.jsx [hunk 1]
```diff
@@ -30,13 +30,16 @@ export default function FileViewCountInline(props: Props) {
   // clean up (only one place edit/remove).
   const isChannelPage = window.location.pathname.startsWith('/@');
 
+  // dont show if no view count, if it's a repost, a livestream or isn't a channel page
   if (!viewCount || (claim && claim.repost_url) || isLivestream || !isChannelPage) {
```
Comment (infinite-persistence) [inline ui/component/fileViewCountInline/view.jsx:34]: Feels like the variables names are already self-explanatory?  There's no additional info here.

**Comment by tzarebczan** [comment]
Let Anythony know linting was failing, otherwise should be good to go.

**Comment by jessopb** [comment]
I think lint is failing because it's a new pattern, and we may not need a new pattern. @infinite-persistence can say whether this optimization is necessary for code splitting?

**Pull request merged**